### DISCUSSION TITLE
Move a distro

### DISCUSSION THREAD

**Discussion by hbadi** [bostrot/wsl2-distro-manager#164] [Ideas]
Hello,

I'd be great to move the distro folder. By defaut wsl install a new distro in the local user folder and when it comes to disk usage/limitation, it'd be great to move a distro to another disk.

Best regards.

**Comment by bostrot** [comment]
Hi,

Good idea! I don't even think this would be very difficult to implement using our WSL API:
Similar to the old copying exporting a distro and importing it with the new location should be sufficient.
